packages/alexa-types/src/version/prototype.ts

```diff
@@ -0,0 +1,13 @@
+import { PermissionType } from '@alexa-types/node';
+import { Product } from '@alexa-types/project';
+
+import { InterfaceType } from '../constants';
+
+/**
+ * Stores survey results in Alexa projects
+ */
+export interface AlexaSurveyContextExtension {
+  products: Record<string, Product>;
+  permissions: PermissionType[];
+  interfaces: InterfaceType[];
```

Comment:
Some channels populate the surveyor context with additional data. The `AlexaSurveyContextExtension` encodes these additional properties for Alexa. There is an extension that covers additional properties from Google and DFES.